app/src/main/java/com/example/bookwormadventuresdeluxe2/SetLocationActivity.java

```diff
@@ -1,10 +1,27 @@
 package com.example.bookwormadventuresdeluxe2;
 
 /**
- * Allows user to pick an exchange location by placing a marker on a map.
```

Comment:
I actually changed this file. Did you resolve the conflict?

Reply:
I updated comments throughout the files SetLocationActvity and ViewLocationActivity. Could you pls pull from master?

Reply:
I must have manually resolved this conflict incorrectly

Reply:
oh no worries!